components/landing/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC } from "react";
+import Image from "next/image";
+
+export interface LandingProps {}
+
+const Landing: FC<LandingProps> = () => {
+  return (
+    <div className="Landing">
+      <div className="Landing__ImageContainer">
+        <img src="/Headshot.jpg" className="Landing__Image" />
+      </div>
+      <div className="Landing__TextContainer">
+        <div className="Landing__Text__Name">Brad Bieselin</div>
+        <div className="Landing__Text__Freelance">-Freelance Web Developer</div>
+        <div className="Landing__Text__Company">-Company: Witbe Inc</div>
+        <div className="Landing__Text__CompanyRole">
+          -Position: Project Manager
+        </div>
+        <div className="Landing__Text__School">
+          -2019 B.S. Computer Science Graduate
+        </div>
+        <div className="Landing__Text__Long">
```

Comment:
Also these need to change.

Reply:
Changed naming convention for consistency